### DISCUSSION TITLE
Route Navigation

### DISCUSSION THREAD

**Discussion by mitraecp** [nuxt/nuxt#20006] [Questions]
Hello, how can i change a child route or a route param, without reload page?

sample

category/10/product/100

```
-category
   -[categoryId]
     -product
       -[productId]
            index.vue
            
```
  in my case, i change the param productId, and entire page is reload. can i update without reload all?

i using this
  
```
  useRouter().replace({
				params: {
				    ...useRoute().params,
				    baseId: project.id
				}
			});
```

**Comment by AloisSeckar** [comment]
Would [navigateTo](https://nuxt.com/docs/api/utils/navigate-to) help you? It helps navigating through the pages without "hard" reload like NuxtLink.
